### DISCUSSION TITLE
How to do a href from q-btn?

### DISCUSSION THREAD

**Discussion by chevyweird** [quasarframework/quasar#12544] [General - Components / Directives / etc] [answered]
` <q-btn  href="https://quasar.dev" target="_blank" class="neonBtn" style="height: 100%; width:80px;" flat >
</q-btn> `

I'm trying to do this as shown in the quasar documentation, but it doesn't work.
Help me please! <3

**Comment by MilosPaunovic** [comment, marked as answer]
[https://codepen.io/milospaunovic/pen/BamrxRw?editors=1011](https://codepen.io/milospaunovic/pen/BamrxRw?editors=1011)

**Reply by chevyweird** [reply]
not working :(

**Reply by MilosPaunovic** [reply]
What isn't working? If you click on a button, you're being redirected to a new page in a new tab. That is what it is supossed to do.

**Reply by chevyweird** [reply]
it's my fault, the quasar version was old. Thank you so much bruh bruh
